### ISSUE TITLE
Add Card component

### ISSUE THREAD

**Issue by pedronauck** [FuelLabs/design-system#3]
- [x] `Card` component
- [x] `Card.Header` component
- [x] `Card.Body` component
- [x] `Card.Footer` component

## Usage

```jsx
import { Card, ButtonLink } from '@fuel-js/react';

export default () => (
  <Card>
    <Card.Header>Some Title</Card.Header>
    <Card.Body>Image a long and beautiful text here!</Card.Body>
    <Card.Footer>
      <ButtonLink variant="link">More info</ButtonLink>
    </Card.Footer>
  </Alert>
);
```

## Custom props

- `boxShadow: BoxShadowSizes`
- `direction: 'row' | 'column'`

**Comment by pedronauck** [comment]
closed on commit 979e6c3421f42e5ef92a2ea0c3d1badf9c5d1102

**Comment by pedronauck** [comment]
closed on commit 979e6c3421f42e5ef92a2ea0c3d1badf9c5d1102
